### ISSUE TITLE
When using an image from an asset, control size of said image

### ISSUE THREAD

**Issue by BarryCarlyon** [jb3rndt/PersistentBottomNavBarV2#20]
https://github.com/BilalShahid13/PersistentBottomNavBar/issues/138#issuecomment-786987458

Since there is actually some maintenance going on.

Do you think you can solve this one?

So for example (under style 3)

```
      PersistentBottomNavBarItem(
        icon: ImageIcon(
          AssetImage('assets/nav/news.png'),
          color: Colors.white,
          size: 512,
        ),
        iconSize: 30,
        title: "News",
        activeColorPrimary: Color.fromRGBO(101, 21, 24, 1),
        inactiveColorPrimary: Colors.grey,
        inactiveColorSecondary: Colors.purple,
      ),
```

Results in a _tiny_ icon and regardless of `size` or `iconSize` the icon on the nav bar is pidly small and will not change size at all

basically looks like what is reported in https://github.com/BilalShahid13/PersistentBottomNavBar/issues/254

**Comment by BarryCarlyon** [comment]
I got a solution via

```
      PersistentBottomNavBarItem(
        icon: Transform.scale(
          scale: 4,
          child: Padding(
            padding: const EdgeInsets.fromLTRB(0.0, 5.0, 0.0, 0.0),
            child: ImageIcon(
              AssetImage('assets/nav/news.png'),
              color: Colors.white,
              size: 512,
            ),
          ),
        ),
        title: "News",
        activeColorPrimary: Color.fromRGBO(101, 21, 24, 1),
        inactiveColorPrimary: Colors.grey,
        inactiveColorSecondary: Colors.purple,
      ),
```

But this seems like a cluster?

**Comment by jb3rndt** [comment]
I'm going to look into it

**Comment by jb3rndt** [comment]
Ok it should be fixed with version 4.1.10.
There was top padding for the tab label in most styles which somehow has no effect to the flutter icons, but did break the layout for the imageIcon.
Feel free to reopen the issue if I missed something

**Comment by BarryCarlyon** [comment]
Can confirm fixed! Praise be!
